docs: PE-5993 (#5562)
spectrocloud/librarium: docs/docs-content/clusters/edge/cluster-management/agent-upgrade-airgap.md

not happen automatically. When you want to upgrade the agent version, you can include the new agent version in a new
cluster profile, and upgrade the cluster using the new profile.

> This page teaches you how to identify the matching agent version to a Palette instance, as well as how to specify a

Reviewer: ```suggestion
This page teaches you how to identify the matching agent version of a Palette instance, as well as how to specify a
```

Palette agent package in the Operating System (OS) pack of a cluster profile. This is useful for upgrading the Palette

> agent on an airgap Edge cluster and for launching new connected clusters but use an older version of the Palette agent.

Reviewer: ```suggestion
agent on an airgap Edge cluster and for launching new connected clusters while using an older version of the Palette agent.
```
